pkg/cluster/internal/meta/meta.go

```diff
@@ -44,6 +45,14 @@ func (c *ClusterMeta) Name() string {
 	return c.name
 }
 
+func (c *ClusterMeta) Network() string {
+	return c.network
+}
+
+func (c *ClusterMeta) SetNetwork(network string) {
```

Comment:
cluster meta is not supposed to be mutable.

I should really clean up how create works :|

Reply:
Ah, I didn't want to add anything to the `NewClusterMeta` func yet since it would require more changes. I can add it as a variadic param to NewClusterMeta since network is not required and existing code doesn't break.

Reply:
er what I really meant by this, is that clusterMeta is a terrible name, this is really just the cluster key (name) and some utils right now, I suspect Create(...) is the right place to plumb this through.